Gemfile

```diff
@@ -41,15 +32,8 @@ group :test do
   end
 
   gem 'mocha', '~> 1.0'
-
-  # Since newer versions of rake are not supported, we pin
-  case RUBY_VERSION
-  when '1.8.7'
-    # No activerecord or sqlite for you
-  else
-    gem 'activerecord', '~> 3.2'
-    gem 'sqlite3'
-  end
+  gem 'activerecord', '~> 3.2'
```

Comment:
This will go away once we remove the legacy activerecord storeconfigs support (soon).